lib/carrierwave/storage/file.rb

```diff
@@ -28,11 +28,8 @@ class File < Abstract
       #
       def store!(file)
         path = ::File.expand_path(uploader.store_path, uploader.root)
-        if uploader.move_to_store
-          file.move_to(path, uploader.permissions, uploader.directory_permissions)
-        else
-          file.copy_to(path, uploader.permissions, uploader.directory_permissions)
-        end
+        file_params = [path, uploader.permissions, uploader.directory_permissions]
```

Comment:
@bensie Just curious if you have any thoughts on this bit of code?  I do my best to make things as simple as possible and feel like this code is helping move in that direction.